### ISSUE TITLE
making an arrow down option for monochrome values

### ISSUE THREAD

**Issue by sarxxt** [publiclab/infragram#299]
![Screenshot (107)](https://user-images.githubusercontent.com/83841462/161126814-fab0fc3a-7676-453d-b618-095553815579.png)

**Comment by sarxxt** [comment]
@TildaDares what you prefer should I open one issue and suggest all changes in it or open new issue for each suggestion.

**Comment by FonyaBrandone** [comment]
Hi @sarxxt I would like to assist you on this if stuck

**Comment by sarxxt** [comment]
> @TildaDares what you prefer should I open one issue and suggest all changes in it or open new issue for each suggestion.

@TildaDares

**Comment by TildaDares** [comment]
Hi @sarxxt, sorry for the late reply. You can open new issues for each suggestion.

**Comment by jywarren** [comment]
Hi @sarxxt this sounds interesting, what do you suggest would be the options available here? Thanks!

**Comment by sarxxt** [comment]
hello @jywarren I think we can add color picker to make it more user friendly. I am attaching some pictures these are the ideas I took from adobe illustrator. Once user selects the color mode we can show these options
![Screenshot (115)](https://user-images.githubusercontent.com/83841462/162140734-a1984896-883f-47ad-bf4e-cdd064cf8163.png)
![Screenshot (114)](https://user-images.githubusercontent.com/83841462/162140750-1e5c9c99-50ae-42b1-83a6-20a560005377.png)

**Comment by jywarren** [comment]
Hi, i think that sounds interesting. Maybe we could use the same module used in image-sequencer:

https://sequencer.publiclab.org/examples/#steps=paint-bucket{}

<img width="499" alt="image" src="https://user-images.githubusercontent.com/24359/163049821-c5c2a50d-d81d-4f1c-97e4-5ece3cf40000.png">

It's a bit complex but you can begin to see how it's integrated (from a library we found) on this line: https://github.com/publiclab/image-sequencer/blob/3caa033c8febf5055c28f616b534933c00fcbe65/examples/lib/defaultHtmlStepUi.js#L110-L120

**Comment by jywarren** [comment]
However, what would the color picker be filling? I can't think of a specific need for a hex color value, what we are doing is kind of the opposite -- generating colors in the R, G, and B fields using some math expressions. Does that make sense?

**Comment by sarxxt** [comment]
yeah. Can we use slider then? like can we translate slider values to math expression, so user wont have to write values?

**Comment by jywarren** [comment]
I wonder, maybe we can offer a menu of different expressions and
descriptions of what they do. But I think a slider can't quite offer the
specific math expressions we'd need!

**Comment by sarxxt** [comment]
@jywarren  I tried adding menu of different expressions in my proposal. I would love to get your feedback on it
